app/src/main/java/ar/com/wolox/android/example/utils/UserSession.kt

```diff
@@ -19,4 +19,15 @@ class UserSession @Inject constructor(private val sharedPreferencesManager: Shar
             field = username
             sharedPreferencesManager.store(Extras.UserLogin.USERNAME, username)
         }
+
+    var useremail: String? = null
```

Comment:
you can replace this for just email. If you want to use user email, you should write it in camelCase: `userEmail`